tests/jeos/firstrun.pm

```diff
@@ -114,6 +114,38 @@ sub verify_partition_label {
     script_output('sfdisk -l') =~ m/Disklabel type:\s+$label/ or die "Wrong partion label found, expected '$label'";
 }
 
+sub create_user_in_terminal {
+    return if script_run("getent passwd $username") == 0;
+
+    assert_script_run "useradd -m $username -c '$realname'";
+    assert_script_run "echo $username:$password | chpasswd";
+}
+
+sub create_users_in_ui {
+    my %args = @_;
+
+    if (exists($args{root}) && !!$args{root}) {
```

Comment:
IMO this should be a separate method entirely

Reply:
I have split the method